CodeEditModules/Modules/AppPreferences/src/Views/AppearenceChangeView.swift

```diff
@@ -0,0 +1,101 @@
+//
```

Comment:
rename the folder `Views` to `HelperViews`

Reply:
<img width="221" alt="CleanShot 2022-03-30 at 20 56 40@2x" src="https://user-images.githubusercontent.com/46642225/160839454-6f0ea88b-249d-417c-9637-dbf979d619f0.png">